app/services/ui_notifications/rss_notifications_checker.rb

```diff
@@ -70,5 +72,19 @@ def rss_notification_blueprint
     def notification_already_exists?(item)
       !!Notification.unscoped.find_by_message(item.summary)
     end
+
+    def load_rss_feed
+      uri = URI.parse(@url)
+      http = Net::HTTP.new(uri.host, uri.port)
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Please do verify SSL or at least provide an option to disable it (but default to verification). We should assume correct SSL stacks and not trust incorrect ones.